assets/_common/styles/custom-styles.scss

```diff
@@ -42,3 +42,10 @@ h1, h2, h3, h4, h5, h6 {
     margin-bottom: 0;
   }
 }
+
+// Widen header title
+@media (min-width: 64em) {
```

Comment:
In place of this `@media` rule we can use the named values from USWDS like
` @include at-media(tablet) {...}` (you can also use `mobile-lg` if you want it to start at a smaller screen)

Reply:
I see what you mean, and got that to work. Thanks for letting me know about these utilities! But now I'm trying to change the original variable so I won't need to use a separate media query.